b2share/modules/records/serializers/schemas/datacite.py

```diff
@@ -76,43 +68,57 @@ class DataCiteSchemaV1(Schema):
     language = fields.Str(attribute='metadata.language')
     alternateIdentifiers = fields.List(
         fields.Nested(AlternateIdentifierSchema, attribute='metadata._pid'))
-    rightsList = fields.List(fields.Function(
-        lambda o: {'rights': o['metadata'].get('licence')}))
+    rightsList = fields.Method('get_rights')
     descriptions = fields.Method('get_descriptions')
 
     def get_creators(self, obj):
-        crs = obj['metadata'].get('creators', ['[Unknown]'])
-        return [{'creatorName':c} for c in crs]
+        crs = obj['metadata'].get('creators', [{'creator_name':'[Unknown]'}])
```

Comment:
Datacite requires at least one creator name, the xml is not valid otherwise. If we make the DOI optional we can remove this hack.